radixdlt-java/src/main/java/com/radixdlt/client/assets/Asset.java

```diff
@@ -52,4 +61,11 @@ public boolean equals(Object o) {
 	public int hashCode() {
 		return iso.hashCode();
 	}
+
+	private boolean isPowerOfTen(int value) {
```

Comment:
I think this method would make more sense to be placed in AssetAmount.java as isPowerOfTen() is too generic of a method for an Asset. An *Asset* cannot be a power of ten but it's subunits property can.